debian/patches/0004-remove-absolute-lib-include-paths.patch

```diff
@@ -0,0 +1,12 @@
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -47,6 +47,6 @@
+ endif()
+ 
+ install(TARGETS smu
+-	ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+-	LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}"
+-	PUBLIC_HEADER DESTINATION "${CMAKE_INSTALL_INCLUDEDIR}")
++	ARCHIVE DESTINATION "lib"
++	LIBRARY DESTINATION "lib"
++	PUBLIC_HEADER DESTINATION "include")
```

Comment:
Shouldn't this already be handled by the GNUInstallDirs include? At least for recent versions of cmake, GNUInstallDirs special cases debian multiarch handling. You'd have to check out 3.0 or whatever version to see if it made it into that version as well.

I'd prefer using this method along with GNUInstallDirs since this change isn't entirely compatible with methodologies used on other distros.